.env.gce.example

```
@@ -0,0 +1,20 @@
+# GCE (Google Compute Engine) Environment Configuration
+# This file is used by docker-compose-gce.yaml
+
+# Django Settings
+DEBUG=True
+DJANGO_ALLOWED_HOSTS=localhost,127.0.0.1
+DJANGO_SETTINGS_MODULE=config.settings.gce
+SECRET_KEY=change-this-to-a-secure-random-key-in-production
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)